### DISCUSSION TITLE
[Feature] Hide Album or or do not display with photos

### DISCUSSION THREAD

**Discussion by eagle470** [immich-app/immich#4729] [Feature Request]
### The feature

I have albums that I would like to not have displayed in the main Photos section. Some are old photo albums of family (grandparents) that I want access to but don't really want to display, others I don't want anyone to see but my wife and I - these I would prefer to password protect and keep out of the main photos gallery.

Switching accounts is a bulky and cumbersome method.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2451
